From c001dff0f6715813bdb8e978dc8c05416c01962b Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@suse.de>
Date: Fri, 16 Dec 2011 13:41:37 -0800
Subject: staging: android: switch: minor code formatting cleanups
Patch-mainline: HEAD
Git-commit: c001dff0f6715813bdb8e978dc8c05416c01962b

This fixes a number of minor space issues in the Android switch code.

Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

diff --git a/drivers/staging/android/switch/switch.h b/drivers/staging/android/switch/switch.h
index 3e4c748..4fcb310 100644
--- a/drivers/staging/android/switch/switch.h
+++ b/drivers/staging/android/switch/switch.h
@@ -30,7 +30,7 @@ struct switch_dev {
 
 struct gpio_switch_platform_data {
 	const char *name;
-	unsigned 	gpio;
+	unsigned	gpio;
 
 	/* if NULL, switch_dev.name will be printed */
 	const char *name_on;
diff --git a/drivers/staging/android/switch/switch_gpio.c b/drivers/staging/android/switch/switch_gpio.c
index 6ba8d97..38b2c2f 100644
--- a/drivers/staging/android/switch/switch_gpio.c
+++ b/drivers/staging/android/switch/switch_gpio.c
@@ -91,7 +91,7 @@ static int gpio_switch_probe(struct platform_device *pdev)
 	switch_data->state_off = pdata->state_off;
 	switch_data->sdev.print_state = switch_gpio_print_state;
 
-    ret = switch_dev_register(&switch_data->sdev);
+	ret = switch_dev_register(&switch_data->sdev);
 	if (ret < 0)
 		goto err_switch_dev_register;
 
@@ -126,7 +126,7 @@ err_detect_irq_num_failed:
 err_set_gpio_input:
 	gpio_free(switch_data->gpio);
 err_request_gpio:
-    switch_dev_unregister(&switch_data->sdev);
+	switch_dev_unregister(&switch_data->sdev);
 err_switch_dev_register:
 	kfree(switch_data);
 
@@ -139,7 +139,7 @@ static int __devexit gpio_switch_remove(struct platform_device *pdev)
 
 	cancel_work_sync(&switch_data->work);
 	gpio_free(switch_data->gpio);
-    switch_dev_unregister(&switch_data->sdev);
+	switch_dev_unregister(&switch_data->sdev);
 	kfree(switch_data);
 
 	return 0;
